### ISSUE TITLE
[Regression] Hyperdimension Neptunia Victory(BLUS31150) crashes after #4611

### ISSUE THREAD

**Issue by Charmareian** [RPCS3/rpcs3#4701]
Hyperdimension Neptunia Victory crashes when entering a dungeon since #4611 

GTX1070
i5 4590

#4635 
![untitled](https://user-images.githubusercontent.com/24770185/40881599-d68560aa-6698-11e8-940e-8753844fbbb3.png)

#4611 
![untitled2](https://user-images.githubusercontent.com/24770185/40881602-e81e1fb4-6698-11e8-93c0-5185beb6af7b.png)

Log from #4611:
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/2065483/RPCS3.zip)

**Comment by kd-11** [comment]
Duplicate of https://github.com/RPCS3/rpcs3/issues/4651
